### ISSUE TITLE
Venous compression spur

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10515]
**Preferred term label:**
Venous compression spur

**Synonyms**
stenosis, band

**Definition (free text, please give PubMed ID)**
Compression of a vein by another structure leading to stenosis

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
May-Thurner syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
if this is May Thurner syndrome, perhaps iot would be clearer to have a term such as  "Iliac vein compression"?

**Comment by MickeySegal** [comment]
The compression and the spurs seem to be separate stages:
https://www.ncbi.nlm.nih.gov/books/NBK554377/
Stage I: asymptomatic left iliac vein compression
Stage II: formation of venous spurs
Stage III: left lower extremity DVT.

But you may be right from a clinical point of view because the data about the spurs is an autopsy finding while the compression is radiological:
"Intravascular ultrasound has greater than 98% sensitivity for iliac vein compression and can characterize the location and severity of fibrosis, stenosis, and thrombus"

So I changed the name of our finding to "Iliac vein compression" and it is up to you whether to have a finding for the venous spurs also.

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6001099
- label: Iliac vein compression
- parents: [Abnormal venous morphology (HP:0002624)](https://hpo.jax.org/app/browse/term/HP:0002624)
- pmid: PMID:32119264
- def: Compression of the common iliac vein against the spine by the contralateral common iliac artery.
- comment: Pulsations of the overriding artery cause shear stress and endothelial changes in the vein, resulting in the formation of fibrous bands or spurs. Over time, the affected area forms a stricture, causing impaired venous return and, in some circumstances, deep venous thrombosis. Common iliac vein compression can be ascertained by Doppler sonography.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10515
- synonyms:
